backend/src/LibExecution/ProgramTypes.fs

```diff
@@ -396,19 +394,19 @@ type Expr =
   | EInfix of id * Infix * Expr * Expr
   // the id in the varname list is the analysis id, used to get a livevalue
   // from the analysis engine
-  | ELambda of id * List<id * string> * Expr
+  | ELambda of id * NEList<id * string> * Expr
```

Comment:
A user might typo and forget to add parameters here. It feels like this should be representable and that we don't want an NEList.

Reply:
do you mean like `List.map ( -> item.prop)`?

Reply:
That sort of feels like "should be parseable in WT, but fail in WT2PT - reminds me of https://github.com/darklang/dark/pull/5013#discussion_r1285275273

Reply:
```elm
List.map (\ -> item.prop)
```